demos/connector/neo4j/directed-graphsage-on-cora-neo4j-example.ipynb

```diff
@@ -110,368 +108,46 @@
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "      <td>0</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "<p>5 rows × 1433 columns</p>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "    0     1     2     3     4     5     6     7     8     9     ...  1423  \\\n",
-       "id                                                              ...         \n",
-       "0      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
-       "1      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
-       "2      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
-       "3      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
-       "4      0     0     0     0     0     0     0     0     0     0  ...     0   \n",
-       "\n",
-       "    1424  1425  1426  1427  1428  1429  1430  1431  1432  \n",
-       "id                                                        \n",
-       "0      0     0     1     0     0     0     0     0     0  \n",
-       "1      0     1     0     0     0     0     0     0     0  \n",
-       "2      0     0     0     0     0     0     0     0     0  \n",
-       "3      0     0     0     0     0     0     0     0     0  \n",
-       "4      0     0     0     0     0     0     0     0     0  \n",
-       "\n",
-       "[5 rows x 1433 columns]"
-      ]
-     },
-     "execution_count": 5,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "node_features = pd.DataFrame(node_data[\"features\"].values.tolist(), index=node_data.index)\n",
-    "node_features.head(5)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "We aim to train a graph-ML model that will predict the \"subject\" attribute on the nodes. These subjects are one of 7 categories:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "{'Case_Based',\n",
-       " 'Genetic_Algorithms',\n",
-       " 'Neural_Networks',\n",
-       " 'Probabilistic_Methods',\n",
-       " 'Reinforcement_Learning',\n",
-       " 'Rule_Learning',\n",
-       " 'Theory'}"
-      ]
-     },
-     "execution_count": 6,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "labels = np.array(node_data[\"subject\"])\n",
-    "set(labels)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "### Splitting the data"
+    "neo4j_graphdb = py2neo.Graph(host=default_host, port=None, user=None, password=None)\n",
+    "neo4j_sg = Neo4jStellarDiGraph(neo4j_graphdb)\n",
+    "neo4j_sg.cache_nodes()"
```

Comment:
I think we should chuck a comment here, explaining whats happening and to not do this for masive graphs!

Reply:
I've added a markdown cell above with a bit more description